pkg/providers/s3/s3_test.go

```diff
@@ -19,10 +19,10 @@ import (
 
 type mockedS3 struct {
 	s3iface.S3API
-
-	Bucket, Key string
-	Output      *s3.GetObjectOutput
-	Error       awserr.Error
+	Error  awserr.Error
```

Comment:
why add this?

Reply:
@aditmeno

Reply:
I haven't added it in, just re-ordered the fields

I ran `golangci-lint --fix` in order for the lint to pass

It relates to `fieldalignment` of govet

Reply:
@aditmeno oh. got it.

Reply:
I've reverted the line back `Bucket, Key string` as how it was previously